pkg/cmd/admin/registry/registry.go

```diff
@@ -131,6 +141,9 @@ func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out io.Writer
 		Long:    registryLong,
 		Example: fmt.Sprintf(registryExample, parentName, name),
 		Run: func(cmd *cobra.Command, args []string) {
+			kcmdutil.CheckErr(cfg.Complete(f, cmd, out, args))
+			kcmdutil.CheckErr(cfg.Validate(cmd, args))
+
 			err := RunCmdRegistry(f, cmd, out, cfg, args)
```

Comment:
I didn't change it because of the `ErrExit` handling below which would change the overall behavior.  I can look at that portion more